source/install/desktop-app-changelog.rst

```diff
@@ -6,12 +6,93 @@ Desktop application changelog
 
 Latest Mattermost Desktop App releases:
 
-- `Release v5.1 <#id1>`_
-- `Release v5.0 <#id6>`_
-- `Release v4.7 <#id22>`_
-- `Release v4.6 <#id40>`_
-- `Release v4.5 <#id52>`_
-- `Release v4.4 <#id69>`_
+- `Release v5.2 <#id1>`_
+- `Release v5.1 <#id3>`_
+- `Release v5.0 <#id18>`_
+- `Release v4.7 <#id34>`_
+- `Release v4.6 <#id52>`_
+- `Release v4.5 <#id64>`_
+
+Release v5.2
+--------------
+
+**v5.2.0 release day: 2022-10-24**
+
+**Download Binaries:** `Mattermost Desktop on GitHub <https://github.com/mattermost/desktop/releases/latest>`_
+
+Compatibility
+~~~~~~~~~~~~~~~
+
+- Desktop Apps are required to be used with any `supported Extended Support Release or a newer Mattermost server version <https://docs.mattermost.com/upgrade/release-lifecycle.html>`_.
+
+Highlights
```

Comment:
We'll want to add Tasos' downloads dropdown under this once it's merged: https://github.com/mattermost/desktop/pull/2227